store/tikv/oracle/oracles/pd.go

```diff
@@ -32,9 +33,12 @@ const slowDist = 30 * time.Millisecond
 
 // pdOracle is an Oracle that uses a placement driver client as source.
 type pdOracle struct {
-	c      pd.Client
-	lastTS uint64
-	quit   chan struct{}
+	c  pd.Client
+	mu struct {
```

Comment:
This lock may decrease concurrency.
Since the `dcLocation` won't change for one TiDB instance, there should only exist 2 scopes for each TiDB instance.

Reply:
What if the user manually changes the `txn_scope` session variable to another DC? According to the proposal, this could happen.